clang/include/clang/Driver/Options.td

```diff
@@ -8024,7 +8024,7 @@ def source_date_epoch : Separate<["-"], "source-date-epoch">,
 // CUDA Options
 //===----------------------------------------------------------------------===//
 
-let Visibility = [CC1Option] in {
+let Visibility = [CC1Option, FC1Option] in {
```

Comment:
This enables 4 CUDA flags in `flang-new -fc1` - this really shouldn't be needed in this PR. I would expect that AMD GPU and CUDA logic to be orthogonal. If that's not the case, then lets try to either fix or at document that.

I appreciate that some sub-optimal behaviour might be inherited from Clang. In such cases, lets try to:
1.  Understand what's going on.
2. If a path towards a fix is clear, lets just fix it.
3. If there's no easy fix, lets document this with a GitHub issue and add a link in the commit/code.

Thanks!

Reply:
PR https://github.com/llvm/llvm-project/pull/99002 removes adding `-fcuda-is-device` option for OpenMP AMDGPU in clang. In consequence we don't need to add `-fcuda-is-device` flag for Flang.